labels-owners.yaml

```diff
@@ -82,6 +88,7 @@ labels:
   area/pipelines:
     owners:
       - ark-kun
+      - elikatsis
```

Comment:
What's the reason for adding @elikatsis here? Is @elikatsis in any of the pipelines OWNERs files or able to speak to pipelines ROADMAP?

The criterion I'm following is who should people ask to identify whether issue X will be fixed. 

@jessiezcc seems like the person driving the roadmap for pipelines.

Reply:
@elikatsis is driving Arrikto's contributions to Kubeflow Pipelines and has delivered a complete design and implementation of PersistentVolume/Snapshot support in the Pipelines SDK.
https://github.com/kubeflow/pipelines/issues/801
https://github.com/kubeflow/pipelines/pull/926

Other than that, he has been actively helping out in closing bugs, as can be seen from the regular stream of PRs:
https://github.com/kubeflow/pipelines/pulls?q=is%3Apr+author%3Aelikatsis+is%3Aclosed

Totally, he has contributed more than 5000 loc.

I believe this is more than enough to be included here.
@elikatsis is the owner of the code for PVC/Snapshot support in Kubeflow Pipelines.

Reply:
I don't doubt that @elikatsis has made significant and impactful contributions to pipelines.

Perhaps we should discuss at a community meeting?

To me the point of listing owner is who is accountable and responsible for maintaing each area going forward. One of the key criterion is knowing who is the owner responsible for every sub-piece in an area (eg. DSL, data versioning, UI, metadata, TFX integration, etc...) so that they can route issues.

My expectation is that right now @jessiezcc how the most visibility and context into KFP as a whole.

For example, if a user reports a problem with the DSL who should I assign that bug to verify if it is in fact  abug, assess priority, and assign someone to fix it?

My sense is that @jessiezcc is the best person for that because she has the most visibility into pipelines as a whole.

My expectation would be that she would have the context to know which areas of KFP are owned/maintained by Arrikto and reassign issues as necessary.

When it comes to KFP, I would defer to @jessiezcc and the other KFP owners to decide who should be listed here. So if you'd like @elikatsis please raise it with here and get her to LGTM it.

Reply:
To elaborate a bit more. The immediate goal of identifying area owners is to identify who can and is responsible for issue triage.

There are currently 236 issues waiting on triage by area owners
https://github.com/orgs/kubeflow/projects/26

Pipelines is under-represented because we don't have the Triage GitHub action running on the pipelines repo.